src/main/scala/org/zalando/spearheads/innkeeper/RouteDirectives.scala

```diff
@@ -33,6 +31,17 @@ trait RouteDirectives {
       }
     }
 
+  def findRouteOwnerTeam(id: Long, routesService: RoutesService, requestDescription: String)(implicit executionContext: ExecutionContext): Directive1[TeamName] =
```

Comment:
you should always try to improve the existing code :)

if you can't comfortably read a line in the github code review, probably that line is too long. Of course, I created lots of long lines myself and I'm not proud of that :)